src/twisted/conch/test/test_keys.py

```diff
@@ -1139,6 +1147,62 @@ def test_blobEd25519(self):
             common.NS(b"ssh-ed25519") + common.NS(publicBytes),
         )
 
+    @skipWithoutEd25519
+    def test_blobSKEd25519(self):
+        """
+        Tests that the correct blob is returned after parse for a sk-ssh-ed25519@openssh.com public key.
+        """
+        sk = DummySK()
+
+        enroll = sk.enroll(
+            SKAlgorithm.ED25519,
+            challenge=b"dummy-challenge",
+            application="ssh:",
+            flags=0x01,  # user presence
```

Comment:
Minor comment.

To make it easier to ready, can we have it as a constant ?
```suggestion
            flags=SK_FLAGS_USER_PRESENCE,
```